openedx/core/lib/api/permissions.py

```diff
@@ -166,4 +166,5 @@ def has_permission(self, request, view):
         return user.is_staff \
             or (user.username == request.GET.get('username')) \
             or (user.username == getattr(request, 'data', {}).get('username')) \
+            or (user.username == getattr(request, 'data', {}).get('user')) \
```

Comment:
@clintonb I'm not clear on why we wouldn't add this here for anyone who adds user to their model.  Let us know why you think this should be in a separate permission file that will get duplicated again and again?  For example, your experiments permissions could have just used this.

Reply:
This is fine.